updateHostsFile.py

```diff
@@ -906,16 +928,18 @@ def minimise_file(input_file, target_ip, output_file):
     for line in input_file.readlines():
         line = line.decode("UTF-8")
 
-        if line.startswith(target_ip):
-            lines.append(line[: line.find("#")].strip() + "\n")
+        if target_ip is None or line.startswith(target_ip):
```

Comment:
I don't think it was desired, there was some empty `\n` with the `minimised` flag, this will fix those